src/components/ListItem/ListItem.scss

```diff
@@ -1,7 +1,38 @@
+@import '../../utils/index';
+
+$small-image-size: 56px;
+$big-image-size: 96px;
+
 .moonstone-listItem {
-    padding: 4px 8px;
+    @include truncate;
+
+    width: 100%;
 
     list-style: none;
+
+    .moonstone-listItem-image_small {
+        flex: 0 0 $small-image-size;
+        height: $small-image-size;
+        margin-right: var(--spacing-small);
```

Comment:
It's not come from your PR, but it miss a spacing between the icon and the label. We always must have a space between the image/icon and the text. So you can add this margin to the Typography element, maybe you can add a class for that.

Reply:
There was a class that was supposed to be applying margin to the left or right of the icon but it wasn't appearing - changed structure a bit and put the icon inside a div so that the margin is applied - need to investigate why the custom classes are showing up as `undefined` for the Icon components later.